genders/fr-FR.json

```diff
@@ -0,0 +1,44 @@
+{
+  "?": "?",
+  "agender": "agenre",
+  "androgyne": "androgyne",
+  "androgynous": "androgyne",
+  "bigender": "bigenre",
+  "cis": "cis",
+  "cisgender": "cisgenre",
+  "decline to provide": "ne souhaite pas le fournir",
+  "f to m": "f to m",
+  "female": "femme",
+  "female to male": "female to male",
+  "femme": "fem",
+  "gender nonconforming": "non-conforme dans le genre",
+  "gender questioning": "en questionnement sur son genre",
+  "gender variant": "genre variable",
+  "genderfluid": "genre fluide",
+  "genderless": "sans genre",
+  "genderqueer": "genderqueer",
+  "intersex": "intersexe",
+  "m to f": "m to f",
+  "male": "homme",
+  "male to female": "male to female",
+  "man": "homme",
+  "neither": "ni l'un ni l'autre",
+  "neutrois": "neutrois",
+  "nonbinary": "non-binaire",
+  "none": "aucun",
+  "other": "autre",
+  "pangender": "pangender",
+  "person": "personne",
+  "polygender": "polygenre",
+  "prefer not to say": "préfère ne pas le dire",
+  "queer": "queer",
+  "questioning": "en questionnement",
+  "trans": "trans",
+  "trans*": "trans*",
+  "transfeminine": "femme/fille trans",
```

Comment:
would it make sense to have "femme trans" and "fille trans" as separate entries?